### ISSUE TITLE
not working

### ISSUE THREAD

**Issue by ItzzNoshima** [34736384/genshin-fps-unlock#395]
hi! that fps unlocker isn't working for me. I've runned it as administrator. I have downloaded Genshin Impact on Epic Games

**Comment by 34736384** [comment]
what's not working? it doesn't run? it doesn't unlock?

**Comment by Confiac1563** [comment]
> what's not working? it doesn't run? it doesn't unlock?

yes, it's doesn't work, i'm using windows 10 and my FPS just locked at 60 :L

**Comment by KimoriWasTaken** [comment]
the unlocker automatically asks for admin, so if it doesn't for you I believe your system already has a shady config. Just press "Start Game" in the unlocker and it'll work, don't launch Genshin via EG.
